config/config.exs

```diff
@@ -33,7 +33,8 @@ config :livebook,
   shutdown_callback: nil,
   storage: Livebook.Storage.Ets,
   update_instructions_url: nil,
-  within_iframe: false
+  within_iframe: false,
+  custom_protocol: ""
```

Comment:
We generally prefer structured config, so let's keep it a list and then join into string when passing to the client :)

```suggestion
  allowed_uri_schemes: []
```